slices/libc6-dev.yaml

```diff
@@ -0,0 +1,135 @@
+package: libc6-dev
+
+essential:
+  - libc6-dev_copyright
+
+slices:
+  copyright:
+    contents:
+      /usr/share/doc/libc6-dev/copyright:
+
+  core-libs:
```

Comment:
We need some further explanation to what this is. During a functional split, `minimum`, `core`, etc., have a meaning, as defined in the chisel-docs. `core-libs` is a new thing. I'm not saying it's wrong, but it begs the question about its purpose and relation to `libs` (if there's even one)

Reply:
talked with @zhijie-yang and proposed a new version. will mark this comment as resolved and re-request review from @clay-lake too